lib/inferno/dsl/auth_info.rb

```diff
@@ -270,7 +270,8 @@ def auth_jwt_header
         {
           'alg' => encryption_algorithm,
           'kid' => private_key['kid'],
-          'typ' => 'JWT'
+          'typ' => 'JWT',
+          'jku' => "#{Inferno::Application['base_url']}/jwks.json"
```

Comment:
Let's add the jwks url as an inferno application property so that no one ever has to build it, like how we do for base url: https://github.com/inferno-framework/inferno-core/blob/main/lib/inferno/config/application.rb#L23